lua/base16-colorscheme.lua

```diff
@@ -4,6 +4,7 @@
 -- https://color.adobe.com/create/color-wheel
 -- http://vrl.cs.brown.edu/color
 
+-- stylua: ignore start
```

Comment:
Some people (me included) have an automatic `stylua` formatting of Lua files and this comment disables it. Having this comment might help future contributors, who would otherwise have to disable automatic formatting.